src/app.config.js

```diff
@@ -3,6 +3,7 @@
 
   angular.module('app.config', [])
     .constant('BASE_URL', 'http://localhost:50000')
+    .constant('LANGUAGE','zh-cn')
```

Comment:
Is there any chance we can ditch the language constants. I think angular translate will do this automatically (https://angular-translate.github.io/docs/#/guide/09_language-negotiation) via: `$translateProvider
  .translations('en', { /* ... */ })
  .translations('de', { /* ... */ })
  .determinePreferredLanguage();`